core/capabilities/ccip/ccipaptos/chain_accessor_factory.go

```diff
@@ -0,0 +1,24 @@
+package ccipaptos
+
+import (
+	"github.com/smartcontractkit/chainlink-ccip/pkg/chainaccessor"
+	"github.com/smartcontractkit/chainlink-ccip/pkg/contractreader"
+	"github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
+	"github.com/smartcontractkit/chainlink/v2/core/capabilities/ccip/common"
+)
+
+// AptosChainAccessorFactory implements cciptypes.ChainAccessorFactory for Aptos chains.
+type AptosChainAccessorFactory struct{}
```

Comment:
This type is already namespaced in `ccipaptos` so it doesn't need the `Aptos` prefix